static/manifests/v1beta1/CustomResourceDefinition/k0s.k0sproject.io_clusterconfigs.yaml

```diff
@@ -428,6 +428,13 @@ spec:
                       autoMTU:
                         description: 'Auto-detection of used MTU (default: true)'
                         type: boolean
+                      extraArgs:
+                        additionalProperties:
+                          type: string
```

Comment:
this is not a valid flag and causes kube-router to error if this property is defined

Reply:
Not sure what you're referring to? The `additionalProperties` stuff is OpenAPI-specific and not referring to actual kube-router flags.